File: lang/java/src/fun/vac/util/TestRunner.java
package fun.vac.util;

public final class TestRunner {

    private static int TestRunnerItemIndex = 0;

    private TestRunner() {}

    public static void parseTest(boolean ok) {
        if (ok) {
            System.out.printf("✓ Item [%d] PASSED%n", TestRunnerItemIndex);
        } else {
            System.err.printf("✗ Item [%d] FAILED%n", TestRunnerItemIndex);
        }

        TestRunnerItemIndex++;
    }
}

File: lang/php/src/fun/vac/util/TestRunner.php
<?php
    namespace fun\vac\util\TestRunner;

    function parseTest(bool $ok): void {
        static $TestRunnerItemIndex = 0;

        if ($ok) {
            printf("✓ Item [%d] PASSED\n", $TestRunnerItemIndex);
        } else {
            fprintf(STDERR, "✗ Item [%d] FAILED\n", $TestRunnerItemIndex);
        }

        $TestRunnerItemIndex++;
    }
?>
